tcl/filter.tcl

```diff
@@ -174,6 +174,14 @@ proc qc::filter_http_request_validate {event {error_handler "qc::error_handler"}
             return [ns_returnbadrequest "\"$url\" is not a valid URL."]
             return filter_return
         }
+
+        set names [qc::form_var_names]
+        foreach name $names {
+            if { [regexp {::} $name] } {
+                log "Invalid form variable name: $name"
```

Comment:
Simply logging the name only. Are we only going to be using this to measure frequency of invalid form variable names or do we wish to log more detail for further investigation?